README.md

```diff
@@ -55,19 +55,19 @@ limitations under the License.
     <a href="https://hf.co/course"><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/course_banner.png"></a>
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as text, vision, and audio. 
+🤗 Transformers provide thousands of pre-trained models to perform tasks on different modalities such as text, vision, and audio.
```

Comment:
🤗 Transformers is a library, it's not "Transformer models", so it's singular.

Pretrained is acceptable as a word and is the convention we chose throughout our codebase (I see 26,690 occurrences across the repository :))